presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcIntegrationSmokeTest.java

```diff
@@ -44,6 +45,12 @@ protected QueryRunner createQueryRunner()
         return createH2QueryRunner(ImmutableList.copyOf(TpchTable.getTables()), properties);
     }
 
+    @Override
+    protected TestTable createTableWithDefaultColumns()
+    {
+        throw new SkipException("requirement not met");
```

Comment:
What kind of a requirement is not met? Please be more specific.

Is there anything preventing testing of the feature with H2?